monai/transforms/smooth_field/dictionary.py

```diff
@@ -0,0 +1,148 @@
+# Copyright 2020 - 2021 MONAI Consortium
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from typing import Any, Dict, Hashable, Mapping, Optional, Sequence, Union
+
+import numpy as np
+
+from monai.config import KeysCollection
+from monai.transforms.smooth_field.array import RandSmoothFieldAdjustContrast, RandSmoothFieldAdjustIntensity
+from monai.transforms.transform import MapTransform, Randomizable, RandomizableTransform, Transform
+from monai.utils import InterpolateMode
+
+__all__ = ["RandSmoothFieldAdjustContrastd", "RandSmoothFieldAdjustIntensityd"]
+
+
+class RandSmoothFieldAdjustContrastd(RandomizableTransform, MapTransform):
+    """
+    Dictionary version of RandSmoothFieldAdjustContrast. The field is randomized once per invocation by default so the
+    same field is applied to every selected key.
+
+    Args:
+        keys: key names to apply the augment to
+        spatial_size: size of input arrays, all arrays stated in `keys` must have same dimensions
+        rand_size: size of the randomized field to start from
+        padder: optional transform to add padding to the randomized field
+        mode: interpolation mode to use when upsampling
+        align_corners: if True align the corners when upsampling field
+        prob: probability transform is applied
+        gamma: (min, max) range for exponential field
+        apply_same_field: if True, apply the same field to each key, otherwise randomize individually
+    """
+
+    def __init__(
+        self,
+        keys: KeysCollection,
+        spatial_size: Union[Sequence[int], int],
+        rand_size: Union[Sequence[int], int],
+        padder: Optional[Transform] = None,
+        mode: Union[InterpolateMode, str] = InterpolateMode.AREA,
+        align_corners: Optional[bool] = None,
+        prob: float = 0.1,
+        gamma: Union[Sequence[float], float] = (0.5, 4.5),
+        apply_same_field: bool = True,
+    ):
+        RandomizableTransform.__init__(self, prob)
+        MapTransform.__init__(self, keys)
+
+        self.trans = RandSmoothFieldAdjustContrast(spatial_size, rand_size, padder, mode, align_corners, 1.0, gamma)
```

Comment:
i would use keyword arguments here. I'm guessing the 1.0 refers to the probability, since we want this to be handled by the parent class, but it would help readability for other users if this were explicit.